packages/react-components/react-carousel/stories/src/Carousel/CarouselDescription.md

```diff
@@ -4,7 +4,7 @@ A `Carousel` component is a sliding window of elements controlled by previous, n
 
 'CarouselNavContainer' offers multiple layouts of the underlying controls which can be suitable for full screen banners, multiple cards within a view, or large image box previews and displays, or use the underlying controls directly for precision placement.
 
-A `CarouselCard` can be full screen, responsive, or partial sizes, it is recommended to enable the `cardFocus` prop on the `CarouselSlider` if the cards are not full width banners (this provided keyboard navigation and accessibility tool access). If cards are intended to be full-screen banners, we recommend relying on the tab index of the internal elements only, while the out-of-view cards will be set to aria-hidden by default to prevent unnecessary tabbing and quick control access.
+A `CarouselCard` can be full screen, responsive, or partial sizes, it is recommended to enable the `cardFocus` prop on the `CarouselViewport` if the cards are not full width banners (this provided keyboard navigation and accessibility tool access). If cards are intended to be full-screen banners, we recommend relying on the tab index of the internal elements only, while the out-of-view cards will be set to aria-hidden by default to prevent unnecessary tabbing and quick control access.
```

Comment:
```suggestion
A `CarouselCard` can be full screen, responsive, or partial sizes, it is recommended to enable the `cardFocus` prop on the `CarouselSlider` if the cards are not full width banners (this provided keyboard navigation and accessibility tool access). If cards are intended to be full-screen banners, we recommend relying on the tab index of the internal elements only, while the out-of-view cards will be set to aria-hidden by default to prevent unnecessary tabbing and quick control access.
```